content/projects/blueocean/index.html.haml

```diff
@@ -21,11 +21,11 @@ links:
           .logo
             = partial('logo.html.haml')
           %h1.intro-title
-            Jenkins Blue Ocean
+            Blue Ocean for Jenkins Pipelines
           %a.btn.btn-secondary{:href => expand_link('doc/book/blueocean')}
             Documentation
-          %a.btn.btn-secondary{:href => expand_link('projects/blueocean/roadmap')}
-            Roadmap
+          %a.btn.btn-secondary{:href => expand_link('projects/blueocean/releases')}
+            Contribute
```

Comment:
Contribute? releases? Is this mixed?

Reply:
Updated by adding another link to the same page with a different label. I don't want to make it two pages since they would be super short.